.travis.yml

```diff
@@ -16,16 +16,19 @@ gemfile:
   - gemfiles/rails_5.0.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
   - gemfiles/mongoid_4.0.gemfile
   - gemfiles/mongoid_5.0.gemfile
   - gemfiles/mongoid_6.0.gemfile
 
 matrix:
-  include:
+  exclude:
+    - gemfile: gemfiles/rails_6.0.gemfile
+      rvm: 2.3
+    - gemfile: gemfiles/rails_6.0.gemfile
+      rvm: 2.4
+  allow_failures:
     - rvm: ruby-head
-      env: RUBYOPT="--enable-frozen-string-literal"
-  # allow_failures:
-  #   - rvm: ruby-head
```

Comment:
From what I can tell, frozen string literals are now the default on ruby-head, which breaks external dependencies.

Reply:
(see also: https://github.com/awesome-print/awesome_print/pull/369#issuecomment-527994310)